rcon/hooks.py

```diff
@@ -59,6 +59,10 @@ def count_vote(rcon: RecordedRcon, struct_log: StructuredLogLine):
 
 
 def initialise_vote_map(rcon: RecordedRcon, struct_log):
+    config = VoteMapConfig()
+    if not config.get_vote_enabled():
```

Comment:
I'd have not put one there though, we'd still want to generate the suggestion, otherwise if you enable the votemap mid map it will not work right away I think. The one in apply_results is enough